applications.json

```diff
@@ -5442,6 +5442,19 @@
         "javascript"
       ],
       "category" : "database"
+    },
+    {
+      "repo_url" : "https://github.com/lm-a/touch-emoji",
+      "title" : "Touch Emoji",
+      "screenshots" : [
+        "https://raw.githubusercontent.com/lm-a/touch-emoji/master/touch-emoji-icon-screenshot.png",
+        "https://raw.githubusercontent.com/lm-a/touch-emoji/master/touch-emoji-recent-screenshot.png"
+      ],
+      "short_description" : "Emoji picker for MacBook Pro Touch Bar. ",
+      "languages" : [
+        "swift"
+      ],
+      "category" : "utilities"
```

Comment:
@ilyalesik I believe we need to create new category for such apps. For example 'touch_bar'? What do you think?

Reply:
I think it's a good idea, although there are not many of them yet. 
MyTouchbarMyRules, Touch Bar Preview, Touch Bar Simulator can be in it.

Reply:
@serhii-londar will you create the new category? Or should I just make changes to PR (categories.json).

Reply:
Yes, please. You can do this in this PR.

Reply:
Done it.